atef/widgets/config.py

```diff
@@ -1428,6 +1431,32 @@ def edit_items(self, old_items: List[str], new_items: List[str]) -> None:
             else:
                 self._edit_item(old, new)
 
+    def _show_context_menu(self, pos):
+        """
+        Displays a context menu that provides copy & remove actions
+        to the user
+
+        Parameters
+        ----------
+        pos : QPoint
+            Position to display the menu at
+        """
+        menu = QMenu(self)
+
+        def copy_selected():
+            items = self.list_strings.selectedItems()
+            if len(items) > 0:
+                QGuiApplication.clipboard().setText(items[0].text(),
+                                                    QClipboard.Mode.Clipboard)
+
+        copy = menu.addAction('&Copy')
+        copy.triggered.connect(copy_selected)
+
+        remove = menu.addAction('&Remove')
+        remove.triggered.connect(self._remove_item_request)
+
+        menu.exec(self.mapToGlobal(pos))
```

Comment:
I think if there are no selected items, the context menu shouldn't be even shown (as they don't do anything)